dask/dataframe/groupby.py

```diff
@@ -87,7 +87,11 @@ def _maybe_slice(grouped, columns):
     """
     Slice columns if grouped is pd.DataFrameGroupBy
     """
-    if isinstance(grouped, pd.core.groupby.DataFrameGroupBy):
+    # generalized approach of groupby checking
+    # cudf and pandas currently do not have much overlap in
+    # attribute functionality
```

Comment:
This is a lot of space for comments to describe something that is, from Dask Dataframe's perspective, pretty fringe.  Most maintainers looking at this won't understand or care much about cudf.